src/compile/scale/component.ts

```diff
@@ -29,15 +27,11 @@ export class ScaleComponent extends Split<ScaleComponentProps> {
   /**
    * Whether the scale definitely includes zero in the domain
    */
-  public get domainDefinitelyIncludesZero() {
+  public domainDefinitelyIncludesZero() {
```

Comment:
lol why do we have to make it no longer a getter?

Reply:
We actually compute something pretty complicated here. Using a getter feels like an antipattern to me.